tasks/configuration.yml

```diff
@@ -1,7 +1,21 @@
 ---
 
+- name: wrap /etc/issue message in cowsay
+  ansible.builtin.shell:
```

Comment:
Any reason this is `shell` and not `command`?

Reply:
Resolved in [bddc8c0](https://github.com/adfinis/ansible-role-motd/pull/5/commits/bddc8c0e0077830e0e60bb08b6f5e80442b4f870)